testing/endtoend/component_handler_test.go

```diff
@@ -137,23 +138,47 @@ func (c *componentHandler) setup() {
 	if config.TestCheckpointSync {
 		appendDebugEndpoints(config)
 	}
-	// Proxies
-	proxies := eth1.NewProxySet()
-	g.Go(func() error {
-		if err := helpers.ComponentsStarted(ctx, []e2etypes.ComponentRunner{eth1Nodes}); err != nil {
-			return errors.Wrap(err, "proxies require execution nodes to run")
-		}
-		if err := proxies.Start(ctx); err != nil {
-			return errors.Wrap(err, "failed to start proxies")
-		}
-		return nil
-	})
-	c.eth1Proxy = proxies
+
+	var builders *components.BuilderSet
+	var proxies *eth1.ProxySet
+	if config.UseBuilder {
```

Comment:
The e2e test setup had to be modified to allow builders to be used. Underneath, builders function as proxies for execution connections, which is why we use it the same way as our `eth1proxy`

Reply:
Don't we still want to connect the existing eth1 client? shouldn't the builder run alongside the existing execution client?